language/En.txt

```diff
@@ -121,7 +121,7 @@ CUSTOM_POG_LIBRARY          = Pog Library
 //Map -> Grid Mode
 MAP_CLEAR                   = Clear map
 MAP_CLEAR_WARNING           = This will clear all lines, pogs, and underlays on the entire layer. Are you sure?
-MAP_GRID_CHANGE             = changes the grid mode.
+MAP_GRID_CHANGE             = %s changes the grid mode.
```

Comment:
Not a deal breaker, but this one constant having formatting does make it inconsistent with the rest.
I wonder if the retrieval of these could be wrapped in a class, something like `getMapGridChange(String playerName)`. Then at least the requirements for each text is clearly documented, even if we only have one use case like this.

I'm going to merge this any way, but something to consider.